### PR TITLE
Fixed #26779 -- Added extra_context parameter to i18n_javascript view.

### PR DESCRIPTION
**Pull request by shabda**
https://code.djangoproject.com/ticket/26779

### CODE REVIEW

**Comment by shabda** [comment]
https://code.djangoproject.com/ticket/26779

File: django/contrib/admin/sites.py [hunk 1]
```diff
@@ -314,9 +314,10 @@ def password_change_done(self, request, extra_context=None):
         request.current_app = self.name
         return password_change_done(request, **defaults)
 
-    def i18n_javascript(self, request):
+    def i18n_javascript(self, request, extra_context=None):
         """
         Displays the i18n JavaScript that the Django admin requires.
+        extra_context is not required here, but added for consistency with rest of admin views.
```
Comment (tobiasmcnulty) [inline django/contrib/admin/sites.py]: Please wrap this line to 79 chars or fewer.

https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style

File: tests/admin_views/tests.py [hunk 2]
```diff
@@ -691,6 +691,13 @@ def test_i18n_language_non_english_fallback(self):
             response = self.client.get(reverse('admin:jsi18n'))
             self.assertContains(response, 'Choisir une heure')
 
+    def test_jsi18n_with_context(self):
+        """
+        Make sure that jsi18n works when the context is applied to all admin urls
```
Comment (tobiasmcnulty) [inline tests/admin_views/tests.py]: "...when `extra_context` is passed to all admin urls."

File: tests/admin_views/urls.py [hunk 3]
```diff
@@ -12,5 +12,7 @@
     url(r'^test_admin/admin4/', customadmin.simple_site.urls),
     url(r'^test_admin/admin5/', admin.site2.urls),
     url(r'^test_admin/admin7/', admin.site7.urls),
+    url(r'^test_admin/admin8/', (admin.site.get_urls(), 'admin', 'admin8'),
+        {"extra_context": dict(foo="bar")}),
```
Comment (tobiasmcnulty) [inline tests/admin_views/urls.py]: Might as well use single quotes here to match the surrounding style.

Also, would this work with `{'extra_context': dict()}`? I don't believe you need to actually pass any values into the context here to test what you want to test.

**Comment by tobiasmcnulty** [comment]
This looks reasonable to me, pending the comments left. After addressing these please squash all commits into one and re-write the commit message per:

- https://docs.djangoproject.com/en/dev/internals/contributing/committing-code/#committing-guidelines
- https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/working-with-git/

**Comment by timgraham** [comment]
Updated per comments and merged in b0acb1e73ef32e368a8a6213e300e66981f976d7, thanks!

**Pull request closed without merging**